src/exampleTypes.js

```diff
@@ -297,43 +297,41 @@ export default F.stampKey('type', {
       },
     },
     onDispatch(event, extend) {
-      // If mutating any group type specific "resetting" keys, set `forceReplaceResponse`
+      // If mutating any row type specific "resetting" keys, set `forceReplaceResponse`
```

Comment:
Interestingly, this probably highlights a bug. We should probably be doing this for both rows and columns (aka actually for groups)

Reply:
I've added the same logic for columns. But I think we don't need that on `onDispatch` logic after this PR is merged in
#182 . If mutation is not specifying the drilldown it's getting reset and this will trigger `shouldMergeResponse` to be false.

Reply:
@daedalus28  I confirm setting `forceReplaceResponse` was just removed. Resetting the drilldown suffices for the purpose